core/domain/base_model_validators.py

```diff
@@ -283,10 +283,22 @@ def _fetch_field_name_to_external_model_references(cls, item):
 
         for external_model_fetcher_details in (
                 cls._get_external_id_relationships(item)):
-            multiple_models_ids_to_fetch[
-                external_model_fetcher_details.field_name] = (
-                    external_model_fetcher_details.model_class,
-                    external_model_fetcher_details.model_ids)
+            if external_model_fetcher_details.field_name == 'committer_ids':
```

Comment:
I wouldn't use the `field_name` as the detection that something should allow `SYSTEM_USERS` rather add a new field to `ExternalModelFetcherDetails` called maybe `allow_system_user_ids` that would filter the `feconf.SYSTEM_USERS`. Also, similar field called `allow_pseudonymous_ids` should be added that would filter IDs that might that are in format defined in https://github.com/oppia/oppia/blob/d76c06b1e49ee8a94117a7c092d5b9f83de40de6/utils.py#L827

Reply:
Done.